### ISSUE TITLE
Prelight tailwind making type submit to be brackground transparent

### ISSUE THREAD

**Issue by kutyel** [matheus23/elm-default-tailwind-modules#10]
Hi! I'm using `elm-land` with `default-tailwind-modules`, and even though I'm correctly setting the bg color for my button type=submit, the background is still transparent! I think this is an issue related to this behaviour in tailwind:
- https://github.com/tailwindlabs/tailwindcss/issues/6602
- https://github.com/tailwindlabs/tailwindcss/discussions/12063
Is there a way to disable preflight mode with this package? 

Here is the relevant `View.elm` file, in case it helps to debug what is wrong:

```elm
toBrowserDocument :
    { shared : Shared.Model.Model
    , route : Route ()
    , view : View msg
    }
    -> Browser.Document msg
toBrowserDocument { view } =
    let
        screen : Html msg
        screen =
            Html.div [] <|
                Css.Global.global Tw.globalStyles
                    :: view.body
    in
    { title = view.title
    , body = [ Html.toUnstyled screen ]
    }
```

Thanks in advance!

**Comment by matheus23** [comment]
What happens if you try doing this:

```elm
toBrowserDocument :
    { shared : Shared.Model.Model
    , route : Route ()
    , view : View msg
    }
    -> Browser.Document msg
toBrowserDocument { view } =
    let
        screen : Html msg
        screen =
            Html.div [] (view.body ++ [ Css.Global.global Tw.globalStyles ])
    in
    { title = view.title
    , body = [ Html.toUnstyled screen ]
    }
```

--- 

Does this also happen without `elm-land`?

**Comment by kutyel** [comment]
The same, I'm not sure if it is elm-land fault but I think the issue is that the global styles are loading after my custom tailwind styles and therefore overriding the thing 😟

@ryannhg any input you could give, is this maybe not the correct way to do tailwind in elm-land? 👀

**Comment by matheus23** [comment]
yeah - unfortunately tailwind working correctly depends on the order that styles are loaded in.
Usually looking at the generated HTML in the browser DevTools should be enough to verify that global styles come first.

**Comment by ad-si** [comment]
I just stumbled upon this problem as well. I'm using it in a normal Elm app without elm-land. What is the correct workaround now?

**Comment by matheus23** [comment]
You need to make sure to load in your global styles before you use them in your app (as in, the global styles element needs to come before html that uses those styles). See this example: https://github.com/matheus23/elm-tailwind-modules/blob/master/test-example/src/Main.elm#L15-L16

This is an unfortunate side-effect of the combination of how elm-css generates CSS, CSS evaluation can depend on declaration order and how tailwind makes use of that.

**Comment by ad-si** [comment]
It is the first element, but I still need `important` overwrites for button background colors:
https://github.com/ad-si/RepoDash/blob/bdfd08aebb07da456a4c03bcc8e454a6f7fb517f/src/View.elm#L38
